configure.ac

```diff
@@ -55,7 +55,7 @@ AC_PROG_INSTALL
 AC_PROG_LN_S
 
 # Automake macros
-AM_PROG_AR
+m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
```

Comment:
Please add a comment indicating that AM_PROG_AR is defined with autoconf version >= 2.69